apps/console/src/features/groups/components/wizard/create-group-wizard-updated.tsx

```diff
@@ -0,0 +1,493 @@
+                dispatch(
+                    addAlert({
+                        description: t("console:manage.features.groups.notifications.createGroup.error.description",
+                            { description: error.response.data.detail }),
+                        level: AlertLevels.ERROR,
+                        message: t("console:manage.features.groups.notifications.createGroup.error.message")
+                    })
+                );
+            } else {
+                closeWizard();
+                dispatch(addAlert({
+                    description: t("console:manage.features.groups.notifications.createGroup.genericError.description"),
+                    level: AlertLevels.ERROR,
+                    message: t("console:manage.features.groups.notifications.createGroup.genericError.message")
+                }));
+            }
+        }).finally(() => {
+            setIsSubmitting(false);
+        })
+        ;
+    };
+
+    /**
+     * Method to handle the create group wizard finish action.
+     *
+     */
+    const handleGroupWizardFinish = (values: any) => {
+        eventPublisher.publish("manage-groups-finish-create-new-group");
+
+        addGroup(values?.BasicDetails);
+    };
+
+    /**
+     * Handles wizard step submit.
+     *
+     * @param values - Forms values to be stored in state.
+     * @param formType - Type of the form.
+     */
+    const handleWizardSubmit = (values: any, formType: WizardStepsFormTypes) => {
+        if (WizardStepsFormTypes.BASIC_DETAILS === formType) {
+            setWizardState({ ...wizardState, [ formType ]: values });
+            handleGroupWizardFinish({ ...wizardState, [ formType ]: values });
+
+        }
+    };
+
+    /**
+     * Function to change the current wizard step to next.
+     */
+    const changeStepToNext = (): void => {
+        switch(currentStep) {
+            case 0:
+                setSubmitGeneralSettings();
+
+                break;
+        }
+    };
+
+    const navigateToPrevious = () => {
+        setPartiallyCompletedStep(currentStep);
+    };
+
+    const handleFinishFlow = () => {
+        setSubmitGeneralSettings();
+    };
+
+    /**
+     * Filters the steps evaluating the requested steps.
+     *
+     */
+    const filterSteps = (steps: WizardStepsFormTypes[]): WizardStepInterface[] => {
+
+        const getStepContent = (stepsToFilter: WizardStepsFormTypes[] | string[]) => {
+
+            const filteredSteps: any[] = [];
+
+            stepsToFilter.forEach((step: WizardStepsFormTypes) => {
+                if (step === WizardStepsFormTypes.BASIC_DETAILS) {
+                    filteredSteps.push(getBasicDetailsWizardStep());
+                }
+            });
+
+            return filteredSteps;
+        };
+
+        if (!requiredSteps) {
+            return getStepContent(steps);
+        }
+
+        return getStepContent(intersection(steps, requiredSteps));
+    };
+
+    /**
+     * Get the wizard basic step.
+     *
```

Comment:
let's add `@returns` TSDoc tag here to state what is returned from this component.

Reply:
Fixed with d08680252dc6b5e81e1864daef86e3756305e27d